src/main.ts

```diff
@@ -231,23 +225,25 @@ async function lintProjects({
     projectLintResultPromiseOutcomes.filter(isPromiseRejectedResult);
 
   outputLogger.logToStdout('');
-  let isAllPassed = true;
+
+  let allPassed = true;
```

Comment:
I changed it because I wanted it to read more like a part of a sentence. So I was thinking of "all passed" like how you'd ask a question ("all passed?"). But maybe that doesn't make sense here. `didAllPass` would probably make more sense, but it's kind of awkward. Ah... I can change it back 😂

Reply:
Okay, done here: [b206993](https://github.com/MetaMask/module-lint/pull/87/commits/b2069936e15862f4c0998a9e86a1e3fc30134a80)